client/src/App.css

```diff
@@ -285,16 +255,16 @@ th{
   margin-top: 125px;
 }
 
-.correct {
-  background-color: #93ff8e;
+.corect {
+  background-color: #89ff9c;
 }
 
-.incorrect {
-  background-color: #ff75c4;
+.incorect {
```

Comment:
```suggestion
.incorrect {
```

Reply:
Hi, I can't fix this change now  because of something happening with my VS code, after merging  this I will make a pull  request and change the spelling